src/components/Toolbar/ToolbarViewSelector.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+const ToolbarViewSelector = ({ children, className, ...rest }) => {
+  const classes = cx('form-group toolbar-pf-view-selector', className);
```

Comment:
do we want the ```...rest``` passed through to the div?

Reply:
yes, added. thanks.